<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemForm</class>
 <widget class="QWidget" name="ItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="picLabel">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileNameLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileSizeLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
